docs/docs/angular/schematics.mdx

```diff
@@ -94,6 +94,21 @@ Alias:
 ng g @datorama/akita:ahes todos
 ```
 
+### Generate a Local Component State
```

Comment:
Thanks for the PR. It looks good. Don't u think it'll be useful to allow the creation of EntityStore instead of a Store?

Reply:
?